FlowerinoPlugin/test/org/flowerplatform/flowerino_plugin/utils/httpclient/HttpTestException.java

```diff
@@ -0,0 +1,24 @@
+package org.flowerplatform.flowerino_plugin.utils.httpclient;
+
+/**
+ * Generic umbrella exception for any errors related to HTTP operations during tests.
+ * 
+ * @author Andrei Taras
+ */
+public class HttpTestException extends Exception {
+
+	private static final long serialVersionUID = -3981337168541768098L;

```

Comment:
de cautat in cele 2 proiecte ale noastre. Nu serializam ac. clase. Deci ar trebui sa adaugam "generated..."